client/app/vps/cloud-database/vps-cloud-database.routing.js

```diff
@@ -3,14 +3,6 @@ import component from './vps-cloud-database.component';
 export default /* @ngInject */($stateProvider) => {
   $stateProvider
     .state('iaas.vps.detail.cloud-database', {
-      resolve: {
-        shouldNotBeCA: /* @ngInject */ (
-          $q,
-          CucFeatureAvailabilityService,
-        ) => CucFeatureAvailabilityService
-          .hasFeaturePromise('VPS', 'cloudDatabase')
-          .then(hasFeature => (hasFeature ? $q.when() : $q.reject('Not authorized'))),
-      },
```

Comment:
Why not keep this, rename it and inject it in parent's `resolve` instead ?

Reply:
This takes complete refactoring of the VPS code. As of now header component is used in a view.  All of this needs to be changed. May be its better to plan this as a complete revamp task?

<header data-ui-view="vpsHeader" data-ng-if="!$ctrl.loaders.init"></header>

 views: {
        'vpsHeader@iaas.vps': {
          component: headerComponent.name,
        },
        'vpsContainer@iaas.vps': {
          component: detailComponent.name,
        },
      },